src/conf_mode/vpn_ipsec.py

```diff
@@ -438,6 +438,12 @@ def verify(ipsec):
                 if 'local_address' in peer_conf and 'dhcp_interface' in peer_conf:
                     raise ConfigError(f"A single local-address or dhcp-interface is required when using VTI on site-to-site peer {peer}")
 
+                tmp = dict_search('options', ipsec)
+                if not tmp or 'disable_route_autoinstall' not in tmp:
+                    print(f'WARNING: When using site-to-site IPsec with VTI '
```

Comment:
Please use: `vyos.base.Warning()` instead as it has build in string formatting.

Reply:
I tried to use it. But it is impossible if I want to use a warning message of more than one line using \n. Or I need to use many spaces.

Reply:
`vyos.base.Warning()` does automatic split and new-line handling after 72 characters

Reply:
Improved version of `vyos.base.Warning()` was merged from https://github.com/vyos/vyos-1x/pull/1660, can you update this PR so we can merge it?

Reply:
done